src/command_modules/azure-cli-interactive/azclishell/app.py

```diff
@@ -479,18 +479,24 @@ def handle_jmespath_query(self, text, continue_flag):
             else:
                 input_dict = self.last.result
             try:
+                cmd_base = text.partition(SELECT_SYMBOL['query'])[0]
```

Comment:
Also, what if there are multiple occurrences of ? in the command?  For example:
`vm show -g {rg} -n {vm}`
`vm delete -g $.resourceGroup -n $.name`

This is a contrived scenario to illustrate the (potential) problem, but I could envision that someone might want to be able to use this syntax more than once.